README.md

```diff
@@ -58,234 +28,116 @@ After you're finished please remove all the comments and instructions!
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
+## 👥 Author <a name="authors"></a>
+👤 Christian Romuald MOMO TONFACK
+- GitHub: [@githubhandle](https://github.com/Momo-87)
+- Twitter: [@twitterhandle](https://twitter.com/Momo_yde)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/christian-momo/)
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] Query and update animals table
+- [ ] Query multiple tables
+- [ ] add "join table" for visits
+- [ ] database performance audit
+- [ ] add database schema diagram
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
-
 ## 🤝 Contributing <a name="contributing"></a>
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
+> Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/momo-87/vet-clinic-database/issues).
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
+>If you like this project just give it a star ⭐️.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
+>I would like to thank Microverse comnunity for their supports.
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
+>This project is [MIT](https://opensource.org/license/mit/) licensed.
```

Comment:
- [ ] The MIT file should be edited to contain your name. You linked to an external MIT file, but since you cannot edit it (I assume), I would recommend you stick to the one provided by [the Microverse template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) which you can edit. Kindly create the MIT file in your root path and link it properly and ensure it works.

  ![image](https://github.com/momo-87/vet-clinic-database/assets/36542986/297acaeb-e1af-4128-b246-f3dee52b959a)